packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -589,6 +589,10 @@ const AccountSwitcher = props => {
         canOpenMulti() &&
         (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
 
+    const can_manage_account_when_virtual =
```

Comment:
Is it possible to combine this into `can_manage_account` and use the `is_virtual` condition there ?

Reply:
@thisyahlen-deriv I'm not sure, because 'can_manage_account' uses in two places without 'can_manage_account_when_virtual' and I suppose this change can be the cause of a new bugs.

Reply:
If thats the case you can create a variable for `(!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency))` since its being repeated many times.